tests/tas-mtls/src/test/java/org/alfresco/rest/MtlsRestTest.java

```diff
@@ -0,0 +1,62 @@
+package org.alfresco.rest;
+
+import java.lang.reflect.Method;
+
+import org.alfresco.rest.core.RestWrapper;
+import org.alfresco.utility.LogFactory;
+import org.alfresco.utility.data.DataContent;
+import org.alfresco.utility.data.DataSite;
+import org.alfresco.utility.data.DataUserAIS;
+import org.alfresco.utility.model.SiteModel;
+import org.alfresco.utility.network.ServerHealth;
+import org.slf4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.testng.AbstractTestNGSpringContextTests;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeSuite;
+
+@ContextConfiguration ("classpath:alfresco-mtls-context.xml")
+public abstract class MtlsRestTest extends AbstractTestNGSpringContextTests
```

Comment:
Do I understand it correctly, that this test is only verifying if Repository is alive and working?